server/db/seeders/20190528204615-demo-user.js

```diff
@@ -10,8 +13,19 @@ export function up(queryInterface, Sequelize) {
         email: 'john.doe@test.com',
         password: 'password',
         signupMethod: 'local',
+        role: 'user',
         socialId: null,
         profilePic: null
+      },
+      {
+          firstName: process.env.SUPER_ADMIN_FIRSTNAME,
```

Comment:
Why is this coming from the environment variable?

Reply:
To protect data?

Reply:
I doubt it matters for seeded users